coderd/database/databasefake/databasefake_test.go

```diff
@@ -0,0 +1,59 @@
+package databasefake_test
+
+import (
+	"fmt"
+	"reflect"
+	"testing"
+
+	"github.com/coder/coder/coderd/database"
+
+	"github.com/coder/coder/coderd/database/databasefake"
+)
+
+// TestExactMethods will ensure the fake database does not hold onto excessive
+// functions. The fake database is a manual implementation, so it is possible
+// we forget to delete functions that we remove. This unit test just ensures
+// we remove the extra methods.
+func TestExactMethods(t *testing.T) {
```

Comment:
This helps find which methods are not required on the fake. Just good for keeping that struct clean.